docs/references/cluster-configuration.md

```diff
@@ -127,18 +118,18 @@ The label name will be `env`, and the value will be what you provide here.
 
 *Value type:* boolean
 
-By default, authentication proof is required when accessing the HTTP API for pushing or querying logs/metrics data.
-This flag can be used to disable said authentication mechanism, allowing for unauthenticated clients to write or read data.
+By default, authentication proof is required when accessing the HTTP API for pushing or querying data.
+This flag disables said authentication mechanism, allowing for unauthenticated clients to write or read data.
```

Comment:
Let's revert this: this is still a boolean parameter, so it can be _used to disable_, but whether or not it really disables depends on the choice of the value.

Reply:
(`data_api_authentication_disabled: false` is the default)

Reply:
I'm trying to eliminate the use of the passive voice.  PTAL.